packages/stark-ui/src/modules/collapsible/collapsible.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule } from "@angular/core";
+import { StarkCollapsibleComponent } from "./components";
+import { MatExpansionModule } from "@angular/material/expansion";
```

Comment:
The path of this import (`@angular/material/expansion`) should also be included in this file: `packages/rollup.config.common-data.js` otherwise Rollup will display a warning for this import while building the Stark packages

Reply:
This is done !